.env.example

```
@@ -13,3 +13,14 @@ NUXT_CLERK_SECRET_KEY=sk_test_...
 NUXT_GOOGLE_CLIENT_SECRET=
+
+# Sentry — https://sentry.io → Settings → Projects → <project> → Client Keys (DSN)
+SENTRY_DSN=https://examplePublicKey@o0.ingest.sentry.io/0
+
+# Sentry source map uploads — https://sentry.io → Settings → Auth Tokens
+# Token needs the `project:releases` scope
+# SENTRY_ORG is your organization slug (visible in the Sentry URL: sentry.io/organizations/<slug>/)
+# SENTRY_PROJECT is your project slug (visible under Settings → Projects → <project>)
+SENTRY_ORG=your-org-slug
+SENTRY_PROJECT=your-project-slug
+SENTRY_AUTH_TOKEN=sntrys_...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SENTRY_AUTH_TOKEN key should go before the SENTRY_ORG key

(UnorderedKey)